packages/block-editor/src/components/block-alignment-control/stories/block-alignment-toolbar.story.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { BlockAlignmentToolbar } from '../';
+
+/**
+ * The `BlockAlignmentToolbar` component is used to render block alignment options in the editor. The different alignment options it provides are `left`, `center`, `right`, `wide` and `full`.
+ */
+const meta = {
+	title: 'BlockEditor/BlockAlignmentToolbar',
+	component: BlockAlignmentToolbar,
```

Comment:
```suggestion
	component: BlockAlignmentToolbar,
	parameters: {
		docs: { canvas: { sourceState: 'shown' } },
	},
```

Let's show the Source panel.